src/MSBuild/CommandLineSwitches.cs

```diff
@@ -265,6 +268,9 @@ private struct ParameterizedSwitchInfo
             new ParameterizedSwitchInfo(  new string[] { "lowpriority", "low" },                ParameterizedSwitch.LowPriority,                null,                           false,          null,                                  true,   false),
             new ParameterizedSwitchInfo(  new string[] { "question", "q" },                     ParameterizedSwitch.Question,                   null,                           false,          null,                                  true,   false),
             new ParameterizedSwitchInfo(  new string[] { "detailedsummary", "ds" },             ParameterizedSwitch.DetailedSummary,            null,                           false,          null,                                  true,   false),
+            new ParameterizedSwitchInfo(  new string[] { "getProperty" },                       ParameterizedSwitch.GetProperty,                null,                           true,           "MissingGetPropertyError",             true,   false),
+            new ParameterizedSwitchInfo(  new string[] { "getItem" },                           ParameterizedSwitch.GetItem,                    null,                           true,           "MissingGetItemError",             true,   false),
+            new ParameterizedSwitchInfo(  new string[] { "getTargetResult" },                   ParameterizedSwitch.GetTargetResult,            null,                           true,           "MissingGetTargetResultError",             true,   false),
```

Comment:
I think that depends on what you see as the primary use case. Are you looking for the json-formatted set that actually takes multiple properties? Or do you see the primary use case to be asking for exactly one property? I don't think one way is obviously more right than the other.

Reply:
Yeah I agree, not sure this needs any change--just mentioning it because I hit it on Item and then, like two minutes later, on TargetResult, so my brain apparently had a very specific expectation.